app/db_models.py

```diff
@@ -345,6 +347,64 @@ def update_obj(self, question, data):
             if field in data:
                 setattr(question, field, data[field])
 
+class FillInTheBlankQuestion(Question):
+    id = db.Column(db.Integer, db.ForeignKey('question.id'), primary_key=True)
+
+    question_text = db.Column(db.String, nullable=False)
+
+    answer = db.Column(db.String, nullable=False)
+
+    __mapper_args__ = {
+        'polymorphic_identity': QuestionType.FILL_IN_THE_BLANK_QUESTION,
+    }
+
+    def get_answer(self):
+        return markdown_to_html(self.answer)
+    
+    def make_question(question_text):
+        answers = {} # dictionary of all the answers and their corresponding text boxes
+        blank = False #determines if the words are the fill in the blank part
+        temp_answer = "" #keeps track of the fill in the blank answer
+
+        new_question = ""
+
+        textbox_number = 1
+
+        for char in question_text:
```

Comment:
It looks like the idea here is to use a ^ to indicate the start of a blank to fill in. Let's make this something a bit more unique (e.g. someone may enter text like 2^x for a math equation in the question). We might change this later, but how about three of them instead of 1 (i.e. ^^^).